recipe/meta.yaml

```diff
@@ -13,11 +13,13 @@ package:
 source:
   # only pull sources after upstream PyPI release...
   url: https://github.com/google/jax/archive/jaxlib-v{{ version }}.tar.gz
-  sha256: ddc14da1eaa34f23430d40ad9b9585088575cac439a2fa1c6833a247e1b221fd
+  sha256: c2c82cd9ad3b395d5cbc0affa26a2938e52677a69ca8f0b9ef9922a52cac4f0c
   patches:
     - patches/0001-Allow-for-custom-CUDA-build.patch
     - patches/0002-xla-Support-third-party-build-of-boringssl.patch
     - patches/0003-xla-Fix-abseil-headers.patch
+    - patches/0004-Omit-usage-of-StrFormat-in-xla.patch
```

Comment:
There are currently redundant (or obsolete) copies of these three patches (2-4) starting with 0001 - please delete.

Reply:
These are all different patches, just the number is the same.

Reply:
OK, that wasn't obvious, as they are sometimes very similarly named, so that the UI chops off the parts that are different, e.g. 
```
recipe/patches/0001-Omit-usage-of-StrFormat.patch
recipe/patches/0004-Omit-usage-of-StrFormat-in-xla.patch
```

What I initially stumbled over the most was that this PR currently contains 8 patches but only actually applies 5 - AFAICT the other three are only useful  as an intermediate quantity for what gets copied into the patch-ception (presumably since the [`third_party/xla`](https://github.com/google/jax/tree/main/third_party/xla) folder isn't actually populated, so we need to let bazel rather than conda apply those patches)?